.travis.yml

```diff
@@ -24,9 +24,4 @@ before_script:
   - sh -e /etc/init.d/xvfb start
   - sleep 3
 script:
-  - npm run lint
-  - npm test
-  - npm run build:prod
-  - npm run server:prod:ci &
-  - sleep 3
-  - npm run e2e
+  - npm run ci
```

Comment:
The complication in running the builds in parallel is that we need two different build directories and setup servers on two different ports to not cause conflicts. With this PR each step is an individual command and `npm run ci` simply composes them. I think parallel builds is a separate PR because it would take some work to setup.

Do you think the Travis builds would be more likely to pass the first time if I added a `sleep 3` between starting the server and starting the test?